templates/ep19/bs/cart/_order_summary.html

```diff
@@ -4,19 +4,27 @@ <h4>Summary</h4>
     <tr>
         <th>code</th>
         <th>ticket</th>
-        <th>&euro;</th>
-        <th class='text-right'>Manage</th>
+        {% if order.is_complete %}
+            <th>&euro;</th>
+            <th class='text-right'>Manage</th>
+        {% else %}
+            <th class='text-right'>&euro;</th>
+        {% endif %}
     </tr>
 
     {% for item in order.orderitem_set.all %}
     <tr>
         <td>{{ item.code }}</td>
         <td>{% if item.ticket %}{{ item.ticket }}{% else %}{{ item.description }}{% endif %}</td>
-        <td>{{ item.price }}</td>
-        <td class='text-right'>
-            <a class='btn btn-outline-primary' href='{% url "user_panel:manage_ticket" item.ticket.id %}'>Configure ticket</a>
-            <a class='btn btn-outline-primary' href='{% url "user_panel:assign_ticket" item.ticket.id %}'>Assign ticket</a>
-        </td>
+        {% if order.is_complete %}
```

Comment:
Could you add a test confirming that the buttons aren't shown when a user checks out?